lib/parallel_report_portal/configuration.rb

```diff
@@ -129,10 +129,17 @@ def load_configuration_file
         .then { |fn| fn ? File.read(fn) : '' }
         .then { |ys| YAML.safe_load(ys, symbolize_names: true) }
         .then do |yaml|
-          ATTRIBUTES.each do |attr|
-            send(:"#{attr}=", yaml[attr]) if yaml&.fetch(attr, nil)
-          end
+        yaml ||= {}
```

Comment:
Small suggestion to improve readability. 
Rest looks good.
```
if yaml
  yaml.transform_keys! { |key| key.downcase }
  ATTRIBUTES.each do |attr|
    yaml_key = yaml.has_key?("rp_#{attr}".to_sym) ? "rp_#{attr}".to_sym : attr
    send(:"#{attr}=", yaml[yaml_key]) if yaml.fetch(yaml_key, nil)
  end
end
```

Reply:
took the 'if yaml' change but kept the rest as is. Benefit is that the block executes considerably faster when yaml is nil(by ms but still a nice win for those not using the config file)